test/single_version/dune

```diff
@@ -13,11 +13,3 @@
    (glob_files *.atd)))
  (action
   (run %{bin:stork} gen %{atd})))
-
-(tests
- (names test_json_parsing)
- (libraries yojson stork test_utils)
- (deps
-  (glob_files *.json))
```

Comment:
dis was unused?

Reply:
hmm actually no, I thought it was at some point and I removed it, but I added a commit to move it back.